q2_quality_control/quality_control.py

```diff
@@ -87,3 +91,25 @@ def evaluate_seqs(output_dir: str, query_sequences: DNAFASTAFormat,
                misclassifications=None, underclassifications=None,
                composition_regression=None, score_plot=None,
                mismatch_histogram=mismatch_histogram, alignments=alignments)
+
+
+def evaluate_taxonomy(output_dir: str, expected_taxa: pd.Series,
```

Comment:
Do we need to add a new citation for this action?

Reply:
Please see my comment above about `pd.Series` vs `pd.DataFrame`.

Reply:
thanks for asking about citations... I modified most of this code from tax-credit so just cited the feature-classifier/tax-credit paper